docroot/modules/custom/sitenow_events/css/single-event.css

```diff
@@ -6,6 +6,10 @@
   }
 }
 
+.sitenow-event .event-field {
```

Comment:
Just noticed that `.sitenow-event` seems like the wrong selector for an event page because it doesn't exist. Changing it to `.sitenow-event-single`, which exists, really borks the display.

Reply:
It seems like the event page looks pretty good now. We may want to have @bspeare take a look and refine it a bit when he gets back, but this looks like a great starting point!